assets/data0_21pure/progs/gametypes/ca.as

```diff
@@ -781,17 +781,23 @@ String @GT_ScoreboardMessage( uint maxlen )
 
             int playerID = ( ent.isGhosting() && ( match.getState() == MATCH_STATE_PLAYTIME ) ) ? -( ent.playerNum + 1 ) : ent.playerNum;
 
+            entry = "";
+            if ( gametype.useSteamAuth )
+            {
+                entry += "&p " + playerID;
+            }
+
             if ( gametype.isInstagib )
             {
                 // "Name Clan Score Ping R"
-                entry = "&p " + playerID + " " + ent.client.clanName + " "
+                entry += "&p " + playerID + " " + ent.client.clanName + " "
```

Comment:
Why `+=` in both cases? now you have `"&p playerID&p playerID"` in your string. (no space between 2nd `&p` and `playerID`